src/Bindings/ManualBindings.cpp

```diff
@@ -4081,6 +4081,32 @@ static int tolua_cCuboid_Move(lua_State * tolua_S)
 
 
 
+static int tolua_cEntity_Destroy(lua_State * tolua_S)
+{
+	// Check the params:
+	cLuaState L(tolua_S);
+	if (!L.CheckParamSelf("cEntity"))
+	{
+		return 0;
+	}
+
+	// Get the params:
+	cEntity * self = nullptr;
+	L.GetStackValue(1, self);
+
+	if (self->IsPlayer())
+	{
+		return L.ApiParamError("Cannot destroy a player");
```

Comment:
You should add a note that the plugin should use `cPlayer:Kick` instead. Or should we even do that directly, rather than throwing an error?

Reply:
The author might want either kicking or respawning. I say let the plugin author decide.

Reply:
Calling `Destroy` to respawn doesn't seem likely to me.
A more likely scenario is a plugin that destroys all entities within some range, so it calls Destroy in a cWorld:ForEachEntity; that includes players.

Reply:
If you kill most entities they are destroyed. If you kill a player it is respawned. In that sense it's analogous.

Reply:
Either way, I don't think we should try and mimic `Destroy` with any other function.

Reply:
Okay, but at least the message to use `Kick` instead should go in. Same as with deprecation, "don't tell them it's the wrong way, tell them what's the right way instead."